src/Symfony/Installer/NewCommand.php

```diff
@@ -36,6 +36,12 @@
 class NewCommand extends Command
 {
     /**
+     * Constants.
+     */
+    const VERSION_LATEST = 'latest';
+    const VERSION_LTS = 'lts';
```

Comment:
I don't see how dealing with this is related to the fact of defining these constants